src/assets/scss/_settings.scss

```diff
@@ -249,11 +249,14 @@ $small-text-font-size: 0.875rem;
 $small-text-line-height: 1.357rem;
 
 $medium-text-font-size: 1.25rem;
-$medium-text-line-height: 1.125;
+$medium-text-line-height: 1.35;
```

Comment:
This change looks like it might affect the text layout or size of the KvDropDownRounded component...did you notice any visual changes there?

Reply:
Outside of this, I didn't see any explicit usages of the medium-text-line-height though it may cascade to the medium breakpoint by default.

Reply:
Good catch, set KvDropdownRounded explicitly to 1.125 to make sure it isn't effected.